doc/nrf/introduction.rst

```diff
@@ -130,34 +111,16 @@ The |NCS| uses :ref:`west <zephyr:west>` to manage the combination of multiple G
 Some of west’s features are similar to those provided by Git Submodules and Google’s Repo tool.
 But west also includes custom features required by the Zephyr project that were not sufficiently supported by the existing tools.
 
-For more details about the reasons behind the introduction of west, see the :ref:`zephyr:west-history` section of the Zephyr documentation.
-
 West's workspace contains exactly one :ref:`manifest repository <zephyr:west-basics>`, which is a main Git repository containing a `west manifest file`_.
 Additional Git repositories in the workspace managed by west are called projects.
 The manifest repository controls which commits to use from the different projects through the manifest file.
 In the |NCS|, the main repository `sdk-nrf`_ contains a west manifest file :file:`west.yml`, that determines the revision of all other repositories.
-This means that sdk-nrf acts as the manifest repository, while the other repositories are projects.
+This means that `sdk-nrf`_ acts as the manifest repository, while the other repositories are projects.
 
 When developing in the |NCS|, your application will use libraries and features from folders that are cloned from different repositories or projects.
 The west tool keeps control of which commits to use from the different projects.
 It also makes it fairly simple to add and remove modules.
 
-Some west commands are related to Git commands with the same name, but operate on the entire west workspace.
-Some west commands take projects as arguments.
-The two most important workspace-related commands in west are ``west init`` and ``west update``.
-
-The ``west init`` command creates a west workspace, and you typically need to run it only once to initialize west with the revision of the |NCS| that you want to check out.
-It clones the manifest repository into the workspace.
-However, the content of the manifest repository is managed using Git commands, since west does not modify or update it.
-
-To clone the project repositories, use the ``west update`` command.
-This command makes sure your workspace contains Git repositories matching the projects defined in the manifest file.
-Whenever you check out a different revision in your manifest repository, you should run ``west update`` to make sure your workspace contains the project repositories the new revision expects (according to the manifest file).
-
-For more information about ``west init``, ``west update``, and other built-in commands, see :ref:`zephyr:west-built-in-cmds`.
```

Comment:
I would keep the old line 159

For more information about the west tool, see the :ref:`zephyr:west` user guide.

Reply:
Line 128 (or 109) also refers to the same user guide, hence removed due to repetition. But can be added if it seems to be relevant their as well.